integration-tests/pino.spec.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable comma-dangle */
+'use strict'
+
+const {
+  FakeAgent,
+  spawnProc,
+  createSandbox,
+  curlAndAssertMessage,
+} = require('./helpers')
+const path = require('path')
+const { assert } = require('chai')
+const { once } = require('events')
+
+describe('pino test', () => {
+  let agent
+  let proc
+  let sandbox
+  let cwd
+  let startupTestFile
+
+  before(async () => {
+    sandbox = await createSandbox(['pino'])
+    cwd = sandbox.folder
+    startupTestFile = path.join(cwd, 'pino/index.js')
+  })
+
+  after(async () => {
+    await sandbox.remove()
+  })
+
+  context('Log injection', () => {
+    beforeEach(async () => {
+      agent = await new FakeAgent().start()
+    })
+
+    afterEach(async () => {
+      proc.kill()
+      await agent.stop()
+    })
+
+    it('Log injection enabled', async () => {
+      proc = await spawnProc(startupTestFile, {
+        cwd,
+        env: {
+          AGENT_PORT: agent.port,
+          lOG_INJECTION: true,
+        },
+        stdio: 'pipe',
+      })
+      const [data] = await Promise.all([
+        once(proc.stdout, 'data'),
+        curlAndAssertMessage(agent, proc, () => {}),
```

Comment:
I think you can replace this line (and the similar one on line 71) with `curl(proc)`, making sure to change the imported function on line 8. This is because we're not making any assertions about the message received by the agent.

Reply:
That makes does make sense. I'll make that change.